swig/swigModule.i

```diff
@@ -168,6 +208,8 @@ namespace COMMON_NS
 	
 		COMMON_NS::DataObjectRepository* getRepository() const;
 	
+		virtual ~AbstractObject();
```

Comment:
I am wondering why it is needed (already asked in a previous comment)

Reply:
See previous comment.